### PR TITLE
Update changelog with recent community PR merges

### PR DESCRIPTION
**Pull request by thisisnotashwin**
- https://github.com/hashicorp/consul-helm/pull/528
- https://github.com/hashicorp/consul-helm/pull/553

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ IMPROVEMENTS:
   StatefulSet and Client Daemonset annotations respectively. This recreates
   the server/client pod when the server/client extraConfig is updated via `helm upgrade` [[GH-550](https://github.com/hashicorp/consul-helm/pull/550)]
 
+* Introduce field server.extraLabels to append additional labels to consul server pods. [[GH-553](https://github.com/hashicorp/consul-helm/pull/553)]
```
Comment (lkysow) [inline CHANGELOG.md:9]:
```suggestion
* Introduce field `server.extraLabels` to append additional labels to consul server pods. [[GH-553](https://github.com/hashicorp/consul-helm/pull/553)]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ IMPROVEMENTS:
...
+
+* Introduce field server.disableFsGroupSecurityContext which disables setting the fsGroup securityContext on the server statefulset.
```
Comment (lkysow) [inline CHANGELOG.md:11]:
```suggestion
* Introduce field `server.disableFsGroupSecurityContext` which disables setting the fsGroup securityContext on the server statefulset.
```

**Review by lkysow — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ IMPROVEMENTS:
   StatefulSet and Client Daemonset annotations respectively. This recreates
   the server/client pod when the server/client extraConfig is updated via `helm upgrade` [[GH-550](https://github.com/hashicorp/consul-helm/pull/550)]
 
+* Introduce field `server.extraLabels` to append additional labels to consul server pods. [[GH-553](https://github.com/hashicorp/consul-helm/pull/553)]
+
+* Introduce field `server.disableFsGroupSecurityContext` which disables setting the fsGroup securityContext on the server statefulset.
+  This enables deploying on OpenShift where the fsGroup is automatically set to an arbitrary gid. [[GH-528](https://github.com/hashicorp/consul-helm/pull/528)]
```
Comment (kschoche) [inline CHANGELOG.md:12]:
```suggestion
  In some environments - eg OpenShift - fsGroup is automatically set to an arbitrary gid so do not try to set it [[GH-528](https://github.com/hashicorp/consul-helm/pull/528)]
```

**Review by kschoche — APPROVED** [review]
Looks great!
Just one minor suggestion.
🥇

**Comment by peterklijn** [comment]
Hi guys, are you planning to do a release soon?

**Comment by kschoche** [comment]
Hi! Yes we are planning a release in the near term with a TBD release date. 
"Coming Soon"

**Pull request merged**